<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="windowTitle">
   <string>替换</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>220</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelFind">
     <property name="text">
      <string>查找目标:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editFind">
     <property name="minimumWidth">
      <number>280</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelReplace">
     <property name="text">
      <string>替换为:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editReplace">
     <property name="minimumWidth">
      <number>280</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupDirection">
     <property name="title">
      <string>方向</string>
     </property>
     <layout class="QHBoxLayout" name="dirLayout">
      <item>
       <widget class="QRadioButton" name="radioUp">
        <property name="text">
         <string>向上(&amp;U)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDown">
        <property name="text">
         <string>向下(&amp;D)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkCase">
     <property name="text">
      <string>区分大小写</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <layout class="QVBoxLayout" name="rightColumn">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnFindNext">
       <property name="text">
        <string>查找下一个(&amp;F)</string>
       </property>
       <property name="minimumWidth">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReplace">
       <property name="text">
        <string>替换(&amp;R)</string>
       </property>
       <property name="minimumWidth">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReplaceAll">
       <property name="text">
        <string>全部替换(&amp;A)</string>
       </property>
       <property name="minimumWidth">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="minimumWidth">
        <number>120</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>